codingstandards/coding-standards-html.md

```diff
@@ -8,7 +8,7 @@
 - <a href="#indentation">Indentation</a>
 - <a href="#miscellaneous">Miscellaneous</a>
```

Comment:
@kowshik-sundararajan Does the `[Miscellaneous](#miscellaneous)` format work? That format doesn't require adding anchors to the main text of the document.